frontend/components/Main/MainHeader/CannotStartAgent.tsx

```diff
@@ -20,7 +20,7 @@ const evictedDescription =
 const AgentEvictedPopover = () => (
   <Popover
     {...otherPopoverProps}
-    title="Your agent was evicted"
+    title="Your agent was banned from work"
```

Comment:
"Banned" implies permanency.. We should use a much less aggressive term here, would check with Oak
```suggestion
    title="Your agent was banned from work"
```

Reply:
I suggested banned, but yes I agree it does sound too permanent.

I suggest "Your agent is suspended from work" instead,